x-pack/plugins/osquery/cypress/e2e/all/packs_create_edit.cy.ts

```diff
@@ -422,58 +421,66 @@ describe.skip('Packs - Create and Edit', { tags: ['@ess', '@serverless'] }, () =
     });
   });
 
-  describe.skip('should open discover in new tab', () => {
-    let packId: string;
-    let packName: string;
+  describe.skip(
```

Comment:
do we want to keep this test skip?

Reply:
I think yes, I didn't intend to fix/unskip this test here.